back/app/policies/project_policy.rb

```diff
@@ -11,9 +45,19 @@ def resolve
       # of the user gives access).
       moderator_scope = user&.active? ? UserRoleService.new.moderatable_projects(user, scope) : scope.none
 
-      moderator_scope
+      moderator_scope = moderator_scope
         .or(resolve_for_visitor)
         .or(resolve_for_normal_user)
+
+      if context[:apply_listed_scope]
+        moderator_scope = ProjectPolicy.apply_listed_scope(
+          moderator_scope,
+          user,
+          context[:include_unlisted] != false
```

Comment:
If `context[:include_unlisted]` is always a boolean then this can just be  `context[:include_unlisted]`